src/v2/Apps/Home/Components/HomeFeaturedArticles.tsx

```diff
@@ -26,53 +30,92 @@ interface HomeFeaturedArticlesProps {
+          <RouterLink
+            key={firstArticle.internalID}
+            to={firstArticle.href ?? ""}
+            style={{ display: "block", textDecoration: "none" }}
+          >
+            {firstImage && (
+              <ResponsiveBox
+                aspectWidth={firstImage.width}
+                aspectHeight={firstImage.height}
+                maxWidth={firstImage.width}
+              >
                 <Image
-                  src={image.src}
-                  srcSet={image.srcSet}
-                  width={325}
-                  height={244}
-                  lazyLoad
-                  alt=""
-                  style={{ display: "block" }}
+                  src={firstImage.src}
+                  srcSet={firstImage.srcSet}
+                  lazyLoad={true}
                 />
-              ) : (
-                <Box bg="black10" width={325} height={244} />
-              )}
+              </ResponsiveBox>
+            )}
 
-              <Spacer mt={2} />
+            <Text variant="xs" textTransform="uppercase" my={1}>
+              {firstArticle.vertical}
+            </Text>
 
-              <Text variant="lg" width={325} pr={1}>
-                {article.title}
-              </Text>
+            <Text variant="xl">{firstArticle.title}</Text>
 
-              {article.author?.name && (
-                <Text variant="lg" width={325} pr={1} color="black60">
-                  {article.author?.name}
-                </Text>
-              )}
+            <Text variant="lg" mt={1}>
+              By {firstArticle.author?.name}
+            </Text>
+          </RouterLink>
+        </Column>
 
-              <Spacer mt={0.5} />
+        <Column span={[12, 6]}>
+          <Masonry columnCount={2}>
+            {articlesList.map((article, index) => {
+              const image = article.thumbnailImage?.small
 
-              <Text variant="sm" color="black60">
-                {article.publishedAt}
-              </Text>
-            </RouterLink>
-          )
-        })}
-      </Shelf>
+              return (
+                <RouterLink
+                  key={article.internalID}
+                  to={article.href ?? ""}
+                  style={{ display: "block", textDecoration: "none" }}
+                >
+                  <Box mb={4}>
+                    {image && (
+                      <>
+                        <ResponsiveBox
+                          aspectWidth={image.width}
+                          aspectHeight={image.height}
+                          maxWidth={image.width}
+                        >
+                          <Image
+                            width="100%"
+                            src={image.src}
+                            srcSet={image.srcSet}
+                            // FIXME: Not sure why second column of masonry grid
+                            // doesn't lazy load. Disabling for now.
+                            // lazyLoad={true}
```

Comment:
Not sure whats up here -- masonry doesn't like it /shrug. It's not really noticeable and only 6 images, but heads up.

Reply:
This should have a `height="100%"` as well; it could be that it's just collapsing to 0 so the intersection observer isn't being triggered.